WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -15805,6 +15828,7 @@ e_armenia = {
 				OR = {
 					ai = no
 					religion_group = christian
+					culture = armenian
```

Comment:
There shouldn't be conditions for de jure titles, but if there must be, the Armenians should be allowed to create their culture's Empire, regardless of their religion. Period.